src/app/learn/get-started/advanced/sticky-settings/index.html

```diff
@@ -81,4 +81,22 @@
     Currently, the <a routerLink="/components/grid">grid component</a> and <a routerLink="/components/tile">tile component</a> both support a <sky-code>settingsKey</sky-code> property for sticky settings. When you provide this property, the components automatically call the <sky-code>SkyUIConfigService</sky-code> stub to retrieve stored settings during appropriate actions. This is why you provide your own implementation as shown above.
   </p>
 
+  <stache-page-anchor>
+    Unit Tests
+  </stache-page-anchor>
+  <p>
+    To avoid external dependencies, unit tests should inject the base class when you provide a custom <sky-code>SkyUIConfig Service</sky-code>.
```

Comment:
Should we tweak the latter part of the sentence to make this a command? I'm thinking "To avoid external dependencies, inject the base class into into unit tests when you..."

Also, it looks like we have a space in `SkyUiConfig Service` that should not be there. At least, there is no space in either instance in the code block that follows.

Reply:
I like that edit much better. Also good eye on the space. Thanks!